.github/workflows/build-and-test.yml

```diff
@@ -1169,6 +1169,18 @@ jobs:
       - uses: actions/download-artifact@v2
         with:
           name: stt-tflite-3.7-aarch64.whl
+      - uses: actions/download-artifact@v2
+        with:
+          name: stt-tflite-3.8-armv7.whl
+      - uses: actions/download-artifact@v2
+        with:
+          name: stt-tflite-3.8-aarch64.whl
```

Comment:
Not sure this is right given we dropped python 3.8

Reply:
Fixed